src/Extracting/ParsesValidationRules.php

```diff
@@ -188,6 +188,12 @@ protected function parseRule($rule, array &$parameterData, bool $independentOnly
                 case 'required':
                     $parameterData['required'] = true;
                     break;
+                case 'accepted':
+                    $parameterData['required'] = true;
+                    $parameterData['type'] = 'boolean';
```

Comment:
This is good, but doesn't the documentation say it can be "yes" or 1? I don't think we can conclusively assign a `boolean` type.

Reply:
Yes, you are right if we think in terms of strict typing. I wanted it to be consistent with the `boolean` [validation rule](https://laravel.com/docs/9.x/validation#rule-boolean), which can be true, false as well as 1, 0, "1" and "0" (but I see there's no "yes" and "no" option). It is still considered a `boolean` **type** in Scribe.

If we leave the type empty, it will default to `string` type, which I don't think makes much sense either, and from what I can see, displaying multiple available types would be pretty hacky.